# Raw Rfans LIDAR packet.
uint32 packNum          # packet number
char[82800] data      # packet contents 600 Block
